SAMLPortal/Client/webpack.config.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5434886)